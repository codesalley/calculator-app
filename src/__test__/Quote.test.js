import renderer from 'react-test-renderer';
import QuotePage from '../components/Quote';

test('render QuotePage properly', () => {
  const res = renderer.create(<QuotePage />).toJSON();
  expect(res).toMatchSnapshot();
});
