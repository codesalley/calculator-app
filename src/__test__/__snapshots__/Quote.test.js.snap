// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render QuotePage properly 1`] = `
<div
  className="quote-main"
>
  <h2
    className="quote-h2"
  >
    Random Quote
  </h2>
  <div
    className="quote-card"
  >
    <img
      alt="page loading"
      className="loading-img"
      src="undraw_Loading_re_5axr.svg"
    />
  </div>
</div>
`;
